packages/story-editor/src/utils/localAutoSave.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function storyKey(storyId, isNew) {
+  return `wp-stories-autosave-story-${isNew ? 'auto-draft' : storyId}`;
+}
+
+export function getLocalAutoSave(storyId, isNew) {
+  return window.sessionStorage.getItem(storyKey(storyId, isNew));
+}
+
+export function setLocalAutoSave(storyId, isNew, story, pages) {
+  window.sessionStorage.setItem(
+    storyKey(storyId, isNew),
+    JSON.stringify({
+      story,
+      pages,
+    })
+  );
+}
+
+export function deleteLocalAutosave(storyId, isNew) {
+  window.sessionStorage.removeItem(storyKey(storyId, isNew));
+}
```

Comment:
Worth doing this similar to our `localStore` API in `packages/design-system/src/utils/localStore.js`?

Note the error handling & error tracking.

Reply:
Moved to design-system in [6959b94](https://github.com/GoogleForCreators/web-stories-wp/pull/12129/commits/6959b94307f65d45164614ca92c960c7ab154c50), followed the same structure for consistency, assuming it might be used for more than one case.